src/Http/ViewComposers/MediasUploaderConfig.php

```diff
@@ -50,9 +50,15 @@ public function compose(View $view)
 
         $mediasUploaderConfig = [
             'endpointType' => $endpointType,
-            'endpoint' => $endpointType === 'local' ? $this->urlGenerator->route('admin.media-library.medias.store') : s3Endpoint($libraryDisk),
+            'endpoint' => $endpointType === 'local'
```

Comment:
This and other ternary should probably be refactored now that we have more than 2 drivers. This is fine for now as we will probably revisit it when migrating away from FineUploader.

Reply:
Should we refactor those now ? 
What's your recommendations on that (helpers functions / switch case...) ?

Reply:
All good for now. Once we refactor the uploader to not use FineUploader anymore, we'll revisit.